pymc3/distributions/multivariate.py

```diff
@@ -687,6 +688,32 @@ def _distr_parameters_for_repr(self):
         return ["n", "a"]
 
 
+class OrderedMultinomial(Multinomial):
+    rv_op = multinomial
```

Comment:
Not sure if this is even needed. I see I put it in the other `Ordered` when I refactored them but it might be unnecessary

Reply:
Apparently it is: I just tried without it and it throws an `AttributeError: 'NoneType' object has no attribute 'ndim_supp'`